src/core/utils.ts

```diff
@@ -3,6 +3,7 @@ import type { Address, BigNumberIsh, Deadline } from './types'
 import Wei from './entities/Wei'
 import BigNumber from 'bignumber.js'
 import invariant from 'tiny-invariant'
+import { TokensPair } from '@/utils/pair'
```

Comment:
Previously I tried to keep `@/utils/pair` await from any other modules... But maybe it's fine.

Reply:
Moved to utils/pair